<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>proposalWindow</class>
 <widget class="QMainWindow" name="proposalWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>proposal info</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget_proposalInfo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>241</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
